### PR TITLE
Navbar navigation links updated

### PR DESCRIPTION
**Pull request by zholochu**
Updated navbar links

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -3,4 +3,7 @@ class PagesController < ApplicationController
 
   def home
   end
+
+  def contact_us
```
Comment (andygw) [inline app/controllers/pages_controller.rb:7]: Will we have a path for this?

**Review by andygw — COMMENTED** [review]
Are we going to be setting up a functional contact flow?

**Comment by zholochu** [comment]
> 
If we have time we will setup actual form which will send an email to pairwise admin.

**Review by andygw — COMMENTED** [review]
Shouldn't be building new user stories as part of the navbar branch (/profile and /contact). Those should be done on their own branch, ideally.

**Comment by andygw** [comment]
> > 
> 
> If we have time we will setup actual form which will send an email to pairwise admin.

Don't add these things to the navbar unless we actually need them. We don't want to spend time on things we might not ever implement. And we don't want the site to have links that don't actually work like the user would expect

**Pull request merged**